lisa/operating_system.py

```diff
@@ -1133,7 +1133,7 @@ def _package_exists(self, package: str) -> bool:
         return False
 
     def _is_package_in_repo(self, package: str) -> bool:
-        command = f"{self._dnf_tool()} --showduplicates list {package} -y"
+        command = f"{self._dnf_tool()} list {package} -y"
```

Comment:
@LiliDeng Can you review if this change has regression? So, it needs to explain parameters in comments. It can help others understand the usage of parameters immediately, instead of checking with original author.

Reply:
@pupacha hi, do you find any issue when use --showduplicates?